src/features/Main/Sidebar/index.jsx

```diff
@@ -0,0 +1,68 @@
+          </svg>
+        </a>
+      </div>
+      <nav className="hidden nav-menu">
+        <ul className="nav-links">
+          <li>
+            <button
+              type="button"
+              className={`${activeTab === 'students' ? 'active' : ''} backbutton`}
+              aria-current="page"
+              onClick={() => handleTabClick('students')}
+            >
+              <span className="icon">
+                <svg xmlnsXlink="http://www.w3.org/1999/xlink" className="icon-24" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M13.9688 14C15.7812 14 17.2188 12.5625 17.2188 10.75C17.2188 8.96875 15.75 7.5 13.9688 7.5C12.1875 7.5 10.75 8.96875 10.75 10.75C10.7188 12.5625 12.1875 14 13.9688 14ZM13.9688 9C14.9375 9 15.7188 9.8125 15.7188 10.75C15.7188 11.7188 14.9375 12.5 13.9688 12.5C13.0312 12.5 12.2188 11.7188 12.2188 10.75C12.2188 9.8125 13.0312 9 13.9688 9ZM20 9C21.375 9 22.5 7.90625 22.5 6.5C22.5 5.125 21.375 4 20 4C18.5938 4 17.5 5.125 17.5 6.5C17.5 7.90625 18.5938 9 20 9ZM15.5312 15H12.4375C9.96875 15 8 16.875 8 19.1875C8 19.6562 8.375 20 8.875 20H19.0938C19.5938 20 20 19.6562 20 19.1875C20 16.875 18 15 15.5312 15ZM9.5625 18.5C9.875 17.375 11.0312 16.5 12.4062 16.5H15.5312C16.9062 16.5 18.0625 17.375 18.4062 18.5H9.5625ZM21.2188 10H19.3125C18.9062 10 18.5312 10.125 18.1875 10.2812C18.1875 10.4375 18.2188 10.5938 18.2188 10.75C18.2188 11.8125 17.8438 12.7812 17.1875 13.5H23.4375C23.75 13.5 24 13.25 24 12.9375C24 11.3125 22.75 10 21.2188 10ZM9.71875 10.75C9.71875 10.5938 9.75 10.4375 9.78125 10.2812C9.4375 10.0938 9.0625 10 8.65625 10H6.75C5.21875 10 4 11.3125 4 12.9375C4 13.25 4.21875 13.5 4.53125 13.5H10.7812C10.125 12.7812 9.71875 11.8125 9.71875 10.75ZM8 9C9.375 9 10.5 7.90625 10.5 6.5C10.5 5.125 9.375 4 8 4C6.59375 4 5.5 5.125 5.5 6.5C5.5 7.90625 6.59375 9 8 9Z" fill="#989BA3" />
+                </svg>
+              </span>
+              <span className="nav-text">Students</span>
+            </button>
+          </li>
+          <li>
+            <button
+              type="button"
+              className={`${activeTab === 'instructors' ? 'active' : ''} backbutton`}
+              aria-current="page"
+              onClick={() => handleTabClick('instructors')}
+            >
+              <span className="icon">
+                <svg xmlnsXlink="http://www.w3.org/1999/xlink" className="icon-24" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could you see if exist a better way to put svg files, for example save them as variables?